.travis.yml

```diff
@@ -56,9 +56,11 @@ script:
  - "if [ \"$NPM_TEST\" = \"1\" ]; then git diff-files --quiet -w --relative=client; fi"
  - "if [ \"$NPM_TEST\" = \"1\" ]; then git diff -w --no-color --relative=client; fi"
  - "if [ \"$NPM_TEST\" = \"1\" ]; then (nvm use $TRAVIS_NODE_VERSION && npm run lint); fi"
+ - "if [ \"$PHPCS_TEST\" = \"1\" ]; then composer run-script lint; fi"
+
+after_success:
  - "if [ \"$PHPUNIT_COVERAGE_TEST\" = \"1\" ]; then vendor/bin/phpunit tests flush=1 --coverage-clover=coverage.xml; fi"
```

Comment:
Why is `vendor/bin/phpunit tests` in after_success? it's the test itself, not ONLY coverage report.